### PR TITLE
[Templating] added the `private` parameter in the TemplateController example

### PR DESCRIPTION
**Pull request by zairigimad**

### CODE REVIEW

File: templates.rst [hunk 1]
```diff
@@ -513,6 +518,9 @@ provided by Symfony:
                     // special options defined by Symfony to set the page cache
                     'maxAge'    => 86400,
                     'sharedAge' => 86400,
+                    
+                    // Whether or not caching should apply for client caches only
```
Comment (OskarStark) [inline templates.rst]:
```suggestion
                    // whether or not caching should apply for client caches only
```

File: templates.rst [hunk 2]
```diff
@@ -494,6 +497,8 @@ provided by Symfony:
                 <!-- special options defined by Symfony to set the page cache -->
                 <default key="maxAge">86400</default>
                 <default key="sharedAge">86400</default>
+                 <!-- Whether or not caching should apply for client caches only -->
```
Comment (OskarStark) [inline templates.rst]:
```suggestion
                 <!-- whether or not caching should apply for client caches only -->
```

File: templates.rst [hunk 3]
```diff
@@ -476,6 +476,9 @@ provided by Symfony:
                 # special options defined by Symfony to set the page cache
                 maxAge:    86400
                 sharedAge: 86400
+                
+                # Whether or not caching should apply for client caches only
```
Comment (OskarStark) [inline templates.rst]:
```suggestion
                # whether or not caching should apply for client caches only
```

**Review by OskarStark — APPROVED** [review]
Thank you, just some minors comments

**Comment by OskarStark** [comment]
One question, is this a new feature introduced in 5.0 or was it just missing?

**Review by HeahDude — APPROVED** [review]
Ok to merge this in 4.4 since this part has been shorten, and we still have a dedicated note in 3.4.

**Comment by HeahDude** [comment]
Thank you! I've merged in 4.4 and applied the cs changes.

**Pull request merged**